packages/cubejs-client-core/index.d.ts

```diff
@@ -680,11 +680,21 @@ declare module '@cubejs-client/core' {
 
   export type TimeDimensionGranularity = 'second' | 'minute' | 'hour' | 'day' | 'week' | 'month' | 'year';
 
-  export type TimeDimension = {
+  export interface TimeDimensionBase {
```

Comment:
Could you please change to be `type`s? While interfaces also work here, our docs-gen (which generates the docs' md files) works better with type definitions rather than with interfaces.

Reply:
Done.